### DISCUSSION TITLE
Does Hide-Video player save bandwidtch?

### DISCUSSION THREAD

**Discussion by amitkuliyal** [pear-devs/pear-desktop#281] [Q&A]
Does the Hide video player plugin save bandwidth by only loading the audio file or it loads the video as well and hides it? I want to confirm this as I mostly use my module data to listen to music.

**Comment by Araxeus** [comment]
It just hide the video.

Sadly the way youtube music works, the audio and the video stream come together (the audio comes from the video)

**Reply by Bellisario** [reply]
Would be very useful to temp download the audio of the video with download library and use it in the player, without use the "hidden" video that consumes a lot of data.
